service/templates/cloudformation/guest/outputs.yaml

```diff
@@ -1,9 +1,13 @@
 {{define "outputs"}}
 Outputs:
-  WorkersOutput:
+  MasterImageID:
+    Value: {{ .MasterImageID }}
```

Comment:
I think we should add outputs for master and worker instance types.

We don't support changing instance types yet but it should be straight forward to implement later. WDYT?

Reply:
Totally, already added, PTAL.